examples/feature_selection/plot_select_from_model_diabetes.py

```diff
@@ -1,100 +1,141 @@
 """
-===================================================
-Feature selection using SelectFromModel and LassoCV
-===================================================
+============================================
+Model-based and sequential feature selection
+============================================
 
-Use SelectFromModel meta-transformer along with Lasso to select the best
-couple of features from the diabetes dataset.
+This example illustrates and compares two approaches for feature selection:
+:class:`~sklearn.feature_selection.SelectFromModel` which is based on feature
```

Comment:
This might be interesting to mention that we use `LassoCV` and the property of the `l1` penalty to select a subset of feature.

Reply:
while that's true, in this case there is only one coefficient that is set to zero by the L1 penalty. So I fear that this might be a little confusing?